cmd/stayrtr/stayrtr.go

```diff
@@ -703,12 +711,17 @@ func (m *metricsEvent) HandlePDU(c *rtr.Client, pdu rtr.PDU) {
 				"_", -1))).Inc()
 }
 
-func (m *metricsEvent) UpdateMetrics(numIPv4 int, numIPv6 int, numIPv4filtered int, numIPv6filtered int, changed time.Time, refreshed time.Time, file string, brkCount int) {
+func (m *metricsEvent) UpdateMetrics(numIPv4 int, numIPv6 int, numIPv4filtered int, numIPv6filtered int, changed time.Time, refreshed time.Time, file string, brkCount int, aspaCount int) {
+	NumberOfObjects.WithLabelValues("vaps").Set(float64(aspaCount))
+	NumberOfObjects.WithLabelValues("bgpsec_pubkeys").Set(float64(brkCount))
+	NumberOfObjects.WithLabelValues("vrps").Set(float64(numIPv4 + numIPv6))
+	NumberOfObjects.WithLabelValues("effective_vrps").Set(float64(numIPv4filtered + numIPv6filtered))
+
+	NumberOfBRKs.WithLabelValues().Set(float64(brkCount))
```

Comment:
It seems like `NumberOfBRKs` never actually got implemented, so feel free to remote it entirely